plugins/sysinfo/sysinfo.cpp

```diff
@@ -338,7 +340,10 @@ getWmiInfo (int mode)
 				hr = pclsObj->Get (L"Name", 0, &vtProp, 0, 0);
 				break;
 		}
-		WideCharToMultiByte (CP_ACP, 0, vtProp.bstrVal, -1, buffer, SysStringLen (vtProp.bstrVal)+1, NULL, NULL);
+		buffer_iter += WideCharToMultiByte(CP_ACP, 0, vtProp.bstrVal, -1, buffer_iter, SysStringLen(vtProp.bstrVal) + 1, NULL, NULL) - 1;
+		*(buffer_iter++) = ',';
```

Comment:
I wouldn't call it "wrong". I was thinking whether I should add one before or after the WideCharToMultiByte call, but I decided to go with after and then remove the last 2 chars after all iterations have been processed.